pcl_ros/src/pcl_ros/features/feature.cpp

```diff
@@ -80,7 +80,7 @@ pcl_ros::Feature::onInit ()
 
   // Enable the dynamic reconfigure service
   srv_ = boost::make_shared<dynamic_reconfigure::Server<FeatureConfig> > (*pnh_);
-  dynamic_reconfigure::Server<FeatureConfig>::CallbackType f = boost::bind (&Feature::config_callback, this, _1, _2);
+  dynamic_reconfigure::Server<FeatureConfig>::CallbackType f = boost::bind (&Feature::config_callback, this, boost::placeholders::_1, boost::placeholders::_2);
   srv_->setCallback (f);
```

Comment:
I think its cleaner if you add `using boost::placeholders::_1` at the top of each file where it is needed. That keeps the diff small

Reply:
done.